NitroxModel/DataStructures/GameLogic/Entity.cs

```diff
@@ -9,7 +11,7 @@ namespace NitroxModel.DataStructures.GameLogic
     [ProtoContract]
     public class Entity
     {
-        public AbsoluteEntityCell AbsoluteEntityCell => new AbsoluteEntityCell(Position, Level);
+        public AbsoluteEntityCell AbsoluteEntityCell => new AbsoluteEntityCell((Position == new Vector3(0, 0, 0)) ? LocalPosition : Position, Level);
```

Comment:
Hmm, is this right?  Should we ever use local position to calculate the AbsoluteEntityCell?  It is probably based off of the parent's position.  Will this mess things up that depend on AbsoluteEntityCell?

Reply:
so far i havent seen any issues with the entitycell being misaligned

Reply:
Just looking at the code, it seems either `Position` _or_ `LocalPosition` is initialized, never at the same time (I haven't checked for external assignments), which must be the reason for this ternary if. Does it make sense to keep them both? This seems somewhat inline with the [split recommendation](https://github.com/SubnauticaNitrox/Nitrox/pull/725/files#r257466085) comment @Sunrunner37 made.

On the same note, would you mind annotating these constructors with their intended usecase? I've been out of the Nitrox loop for a while, and can't easily figure out where either is supposed to be used.